src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import _ from 'lodash';
+import classNames from 'classnames';
+import Pagination from '@department-of-veterans-affairs/formation/Pagination';
+import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
+import Scroll from 'react-scroll';
+import { connect } from 'react-redux';
+import {
+  searchInputChange,
+  selectInstitution,
+  searchSchools
+} from '../complaint-tool/actions/schoolSearch';
+import {
+  selectCurrentPageNumber,
+  selectInstitutionQuery,
+  selectInstitutions,
+  selectInstitutionSelected,
+  selectPagesCount,
+  selectSearchInputValue,
+  selectSearchResultsCount,
+  selectShowInstitutions,
+  selectShowInstitutionsLoading,
+  selectShowPagination,
+  selectShowPaginationLoading
+} from '../complaint-tool/selectors/schoolSearch';
+
+const Element = Scroll.Element;
+const scroller = Scroll.scroller;
+
+export class SchoolSelectField extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.debouncedSearchInstitutions = _.debounce(
+      value => this.props.searchSchools(value),
+      150);
```

Comment:
It looks like we will want to cancel this in `componentWillUnmount`, according to `https://github.com/facebook/react/issues/1360#issuecomment-348090222`

Reply:
good catch- updated